machine/meta-xt-images-generic-armv8/conf/machine/generic-armv8-xt.conf

```diff
@@ -2,7 +2,7 @@
 #@NAME: generic armv8 machine
 #@DESCRIPTION: Machine configuration for running a generic armv8
 
-require conf/machine/include/arm/arch-armv8.inc
```

Comment:
this will break building with Yocto < zeus?

Reply:
I'm not author of commit, but will give answer, just to record info.

Commit eba9c1be73a8c1567795b2073fa850537336c7f6 in poky made file arch-armv8.inc just a wrapper over arch-arm64.inc.
arch-armv8.inc left for compatibility reasons and all required definitions were moved to arch-arm64.inc